cmd/neofs-cli/modules/container/list_objects.go

```diff
@@ -73,7 +73,15 @@ var listContainerObjectsCmd = &cobra.Command{
 				if err == nil {
 					attrs := resHead.Header().UserAttributes()
 					for i := range attrs {
-						cmd.Printf("  %s: %s\n", attrs[i].Key(), attrs[i].Value())
+						key := attrs[i].Key()
+						val := attrs[i].Value()
+
+						if key := attrs[i].Key(); key == object.AttributeTimestamp {
```

Comment:
```suggestion
						if key == object.AttributeTimestamp {
```

duplicated key getting

Reply:
fixed